api/InputDeviceInfo.json

```diff
@@ -20,7 +20,7 @@
             "version_added": null
           },
           "ie": {
-            "version_added": null
+            "version_added": false
```

Comment:
`InputDeviceInfo` is something returned by `navigator.mediaDevices.enumerateDevices`, which IE11 doesn't support, confirmed by http://mdn-bcd-collector.appspot.com/tests/api/MediaDevices/enumerateDevices, and already set in https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/enumerateDevices#Browser_compatibility